components/FoodCard.js

```diff
@@ -0,0 +1,17 @@
+import Image from "next/image";
+import Link from "next/link";
+import React from "react";
+import styled from "styled-components";
+import salmon from "../assets/images/salmon.jpg";
+
+export default function FoodCard({ data }) {
+  return (
+    <Card>
+      <Image src={salmon} width={200} alt="salmon with zucchini" />
+    </Card>
+  );
+}
```

Comment:
In this component you receive data (line 7) but then you do not use it. Instead the image source is hard-coded, meaning that always the same image will be rendered.

```suggestion
export default function FoodCard({ recipe }) {
  return (
    <Card>
      <Image src={recipe.pic} width={200} alt={recipe.name} />
    </Card>
  );
}
```